### ISSUE TITLE
Add nested accessors under the ban rule?

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [palantir/tslint#2821]
### Bug Report

- __TSLint version__: 5.3.2
- __TypeScript version__: 2.3
- __Running TSLint via__: (pick one) VSCode

#### TypeScript code being linted

```typescript
chai.assert.equal(foo, bar); // should be strictEqual instead
```

with `tslint.json` configuration:

```json
{
    "rules": {
        "ban": [
            true,
            ["chai.assert", "equal", "Use strictEqual (===) instead of equal (==)."],
            ["chai", "assert.equal", "Use strictEqual (===) instead of equal (==)."]
        ]
    }
}
```

#### Actual behavior

No errors.

#### Expected behavior

Either should complain about using `equal` instead of `strictEqual`.

**Comment by ajafff** [comment]
linking discussion in my PR https://github.com/palantir/tslint/pull/2547#issuecomment-293895511
